releases/release-1.19/README.md

```diff
@@ -76,6 +76,7 @@ The 1.19 release cycle is proposed as follows:
 | **Cherry Pick Deadline** (EOD PST) | Branch Manager | Thu, August 6 | week 17 | |
 | **Test Freeze** (EOD PST) | Branch Manager | Thu, August 6 | week 17 | |
 | **KubeCon CloudNativeCon Europe  -  Break** | Community | Mon, August 10 | week 18 | |
+| **[Code Thaw]** (EOD PST) | Branch Manager | Thu, August 18 | week 19 | |
```

Comment:
I't now directly in the KubeCon break, do we wanna move it before the break?

Reply:
I say yes to moving before the break

Reply:
Sounds good!